ehsm_kms_service/ehsm_napi.js

```diff
@@ -280,7 +280,33 @@ const ehsm_napi = ffi.Library('./libehsmnapi', {
   NAPI_RA_GET_API_KEY: ['string', ['string']],
   NAPI_Enroll: ['string', []],
   NAPI_GenerateQuote: ['string', ['string']],
-  NAPI_VerifyQuote: ['string', ['string', 'string']],
+  /**
+  NAPI_VerifyQuote
+  Description:
+    Users are expected already got a valid DCAP format QUOTE. And it could use this API to send it to eHSM-KMS to do a quote verification.
+  params:
+    - quote_base64:
+        des: A valid DCAP quote in BASE64 string.
+        type: string
+    - mr_signer:
+        des: Trusted zone code issuer public key metric

```

Comment:
what does the des mean? description?  generally, des normally stands for destination...

you can use "desc" if you want.

and also, mr_signer stores the hash value of the enclave author’s public key
mr_enclave stores the hash value of the enclave measurement

Reply:
will be change